src/simtools/applications/run_application.py

```diff
@@ -49,6 +67,12 @@ def _parse(label, description, usage):
         required=True,
         default=None,
     )
+    config.parser.add_argument(
+        "--steps",
+        type=int,
+        nargs="+",
+        help="List of steps to be execution (e.g., '--steps 7 8 9'; ignore to run all).",
```

Comment:
I suggest to write either "do not specify to run all" or remove that comment entirely. As is it is a bit confusing to me.

Reply:
Good suggestion, changed it.